provider/dynamodb.go

```diff
@@ -275,6 +276,108 @@ func (store *dynamodbOnlineStore) CheckHealth() (bool, error) {
 	return true, nil
 }
 
+func (store *dynamodbOnlineStore) ImportTable(feature, variant string, valueType ValueType, source filestore.Filepath) (ImportID, error) {
+	fmt.Printf("Checking metadata table for existing table %s\n", GetTablename(store.prefix, feature, variant))
+	_, err := store.GetFromMetadataTable(GetTablename(store.prefix, feature, variant))
+	if err == nil {
+		return "", &TableAlreadyExists{feature, variant}
+	}
+	fmt.Printf("Updating metadata table %s\n", GetTablename(store.prefix, feature, variant))
+	err = store.UpdateMetadataTable(GetTablename(store.prefix, feature, variant), valueType)
+	if err != nil {
+		return "", err
+	}
+
+	fmt.Printf("Building import table input for %s\n", GetTablename(store.prefix, feature, variant))
```

Comment:
out of the scope for this but we should just have loggers and avoid the print statements haha

Reply:
I agree. For some things (e.g. `Coordinator`) we do, and I like using them; however, for other components (e.g. providers) we haven't taken that step.